src/core/Akka/IO/SocketChannel.cs

```diff
@@ -96,6 +98,31 @@ public virtual bool IsOpen()
         /// <returns>TBD</returns>
         public bool Connect(EndPoint address)
         {
+#if CORECLR
```

Comment:
This is a temporary solution. Waiting for this https://github.com/akkadotnet/akka.net/pull/2405